sdk/tables/Azure.Data.Tables/src/TableItem.cs

```diff
@@ -8,9 +9,15 @@ namespace Azure.Data.Tables.Models
     public partial class TableItem
     {
         internal string OdataType { get; }
```

Comment:
nit:  Should we add a summary here just so this poor member doesn't feel left out?  It seems to be the only one without.